pkgs/tools/package-management/lix/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib
```

Comment:
Minor, but I think [all new packages are supposed to go by in `by-name`](https://github.com/NixOS/nixpkgs/blob/ec9c5bdaf8233cf4e13f51e1ac523183492dd792/pkgs/README.md) now, (and RFC formatted? that seems to be the convention but I couldn't find it written down)

Reply:
Multi-version packages doesn't have a `by-name` version. But yep, we can apply the new RFC for the formatting.